src/components/ActivityScore/Icon/Icon.component.tsx

```diff
@@ -38,7 +40,7 @@ interface IconProps {
 }
 
 const Icon = ({ type, size = 44 }: IconProps) => {
-  const icon = React.cloneElement(scoreIcon[type], {
+  const icon = React.cloneElement(scoreIcon[type] ?? <svg />, {
```

Comment:
예외케이스에서 svg를 렌더링 해주는 이유가 혹시 뭔가요~?

Reply:
@HaJunRyu 스타일이 밀리지 않을까 했는데 td 안이라 안 밀리는 것 같네요
이렇게 변경할까요?

```suggestion
 if (!scoreIcon[type]) return null;

  const icon = React.cloneElement(scoreIcon[type], {
```

Reply:
@HaJunRyu required prop이긴 한데 런타임에 서버에서 임의로 추가한 type이 들어오면 cloneElement에서 Error를 던져버려서 Uncaught Error로 아예 죽어버리는 문제가 있었어요.